src/Controllers/SuperadminController.php

```
@@ -169,6 +172,93 @@ public function subscriptions(): void
+
+    public function createPackage(): void
+    {
+        try {
+            Package::create($this->db, $this->packageFormData());
+            $_SESSION['flash_message'] = 'Package created.';
+        } catch (\Throwable $e) {
+            $_SESSION['flash_error'] = 'Package could not be created: ' . $e->getMessage();
+        }
+
+        $this->response->redirect('/superadmin/packages');
+    }
```

PHPMD (2.15.0)

[error] createPackage accesses the super-global variable $_SESSION. (undefined)

(Superglobals)